tests/lib/rules/comma-dangle.js

```diff
@@ -271,7 +271,7 @@ ruleTester.run("comma-dangle", rule, {
             options: ["always-multiline"],
         },
         {
-            code: "foo(\na,\mb\m)",
```

Comment:
Actually, might as well leave the new version, as there's no other test that tests `foo(\na,b)` (as in, newline after opening parenthesis, no newline after first argument)